java/src/org/openqa/selenium/remote/RemoteWebDriver.java

```diff
@@ -175,16 +175,20 @@ private static URL getDefaultServerURL() {
   }
 
   private static CommandExecutor createExecutor(URL remoteAddress, boolean enableTracing) {
```

Comment:
we shouldn't be needing any change in RemoteWebDriver. Can you please help revert all the changes in this class ?

Reply:
@code-with-abdullah I agree with @krmahadevan, please revert the changes to this class.